capa/features/extractors/dnfile/extractor.py

```diff
@@ -68,9 +69,10 @@ def get_type(self, token: int) -> Optional[Union[DnType, DnUnmanagedMethod]]:
 
 
 class DnfileFeatureExtractor(FeatureExtractor):
-    def __init__(self, path: str):
+    def __init__(self, path: str, min_len: int = DEFAULT_STRING_LENGTH):
```

Comment:
```suggestion
    def __init__(self, path: str, min_str_len: int = DEFAULT_STRING_LENGTH):
```

Reply:
there's not enough context here to infer "min_len" of what? so lets include "str" in the variable/property names

Reply:
Since dnfile/file.py uses functions from dotnetfile.py, I changed `extract_features` in dnfile/file.py to alter the `ctx` variable for the dotnetfile functions ( `ctx = {"min_len": ctx["min_str_len"]}` ). I didn't change the other functions in dnfile/file.py to make them more concise. But I'm not sure if the inconsistency would be confusing?